images/httpbin/rootfs/Dockerfile

```diff
@@ -17,9 +17,14 @@ FROM alpine:3.11
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-RUN apk add --no-cache \
-    python3 python3-dev musl-dev gcc g++ make libffi libffi-dev libstdc++ \
- && pip3 install gunicorn httpbin gevent \
+RUN echo "@edge http://nl.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
```

Comment:
how's this improving the build time?

Reply:
By half, installing alpine packages instead of using pip
**Times:**
- [new](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/ingress-nginx/5324/pull-ingress-nginx-e2e-1-18/1252997042658086912#1:build-log.txt%3A1133)
- [old](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/ingress-nginx/5324/pull-ingress-nginx-e2e-1-18/1250194309513220096#1:build-log.txt%3A1069)